dotnet/src/SemanticKernel.Abstractions/Data/RecordDefinition/IndexKind.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Data;
+
+/// <summary>
+/// Defines the index types that can be used to index vectors.
```

Comment:
Same as previous comment, do we need to make this clear these are well known types but additional types can be added

Reply:
Yep, will reword to clarify.